### PR TITLE
Try parallel Sphinx builds and hide progress

### PR DESCRIPTION
**Pull request by wouterj**
Let's see if we can make Sphinx faster this way :)

### CODE REVIEW

**Comment by smoench** [comment]
I also tried this before and got the same result. But I wasn't able to fix https://github.com/fabpot/sphinx-php

**Comment by wouterj** [comment]
@smoench you're right, I've submitted a "fix" to the sphinx extensions: https://github.com/fabpot/sphinx-php/pull/43

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
                 run: pip install -r _build/.requirements.txt
 
             -   name: "Build documentation"
-                run: make -C _build SPHINXOPTS=-nW html
+                run: make -C _build SPHINXOPTS="-nqW -j2" html
```
Comment (fabpot) [inline .github/workflows/ci.yaml]: Why about using `-j auto` instead?

Comment (fabpot) [reply .github/workflows/ci.yaml]: Answer: because we are still using Sphinx 1.4.9 :)

**Comment by OskarStark** [comment]
I restarted the build

**Comment by smoench** [comment]
The commit hash in `_build/.requirements.txt` needs to be updated

File: _build/conf.py [hunk 2]
```diff
@@ -35,7 +35,7 @@
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
     'sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.todo',
-    'sensio.sphinx.refinclude', 'sensio.sphinx.configurationblock', 'sensio.sphinx.phpcode', 'sensio.sphinx.bestpractice', 'sensio.sphinx.codeblock',
+    'sensio.sphinx.configurationblock', 'sensio.sphinx.phpcode', 'sensio.sphinx.bestpractice', 'sensio.sphinx.codeblock',
     'symfonycom.sphinx'
```
Comment (smoench) [inline _build/conf.py]: This extension `symfonycom.sphinx` (`_build/_themes/_exts/symfonycom/`) needs also be defined as `parallel_read_safe`

**Comment by wouterj** [comment]
Sadly, seems like Sphinx v1.4.9 has a bug with parallel runs (the feature was also marked as "experimental in 1.4"). So back to non-parallel runs, but now with a less verbose output in the GitHub Action.

**Pull request merged**